rfc-099-stack-overflow-for-gds.md

```diff
@@ -0,0 +1,41 @@
+# Q&A Forum for GDS developers
+
+## Summary
+
+A searchable, long-lived body of developer knowledge would make finding
+answers easier.
+
+StackOverflow is a Q&A forum that all developers at GDS are familiar with.
+A private instance of StackOverflow would ensure that valuable knowledge
+is retained and more easily found.
+
+## Problem
+
+When faced with a problem we're not able to solve, we ask for help from
+other developers in person or over Slack.
+
+Both of these methods mean that developer knowledge is quite often lost,
+and time is spent finding answers to the same problems. The solutions
+remain in Slack, and aren't easily retrievable.
+
+## Proposal
+
+I propose GDS uses [StackOverflow for Teams](https://stackoverflow.com/teams).
+
+As developers, we all know the value of StackOverflow. I've used their Teams
+product before and have found it very useful.
+
+StackOverflow allows for org-specific tagging (e.g. `publishing-api`).
+
+Developers can gain karma, and be more easily recognised for helping others.
```

Comment:
I'm a little bit wary of how this leads to in-group/out-group stuff on SO proper, so as long as we can avoid importing that culture it's probably fine to have the karma system.

Reply:
I agree! Hopefully people will ask/answer questions in the same manner as they do on Slack.

Reply:
Agree with Elliot, not sure we should be adopting something built by people who do not value healthy diverse communities.